apiserver/facades/client/modelmanager/package_test.go

```diff
@@ -9,11 +9,11 @@ import (
 	"github.com/juju/juju/internal/testing"
 )
 
-//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/common_mock.go github.com/juju/juju/apiserver/common BlockCheckerInterface
-//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/domain_mock.go github.com/juju/juju/apiserver/common ControllerConfigService,BlockCommandService
-//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/migrator_mock.go github.com/juju/juju/apiserver/facades/client/modelmanager ModelExporter
-//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/service_mock.go github.com/juju/juju/apiserver/facades/client/modelmanager ApplicationService,AccessService,SecretBackendService,ModelService,DomainServicesGetter,ModelDefaultsService,ModelInfoService,ModelConfigService,NetworkService,ModelDomainServices,MachineService,ModelAgentService
-//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/status_mock.go github.com/juju/juju/apiserver/facades/client/modelmanager ModelStatusAPI
+//go:generate go run go.uber.org/mock/mockgen -typed -package modelmanager_test -destination common_mock_test.go github.com/juju/juju/apiserver/common BlockCheckerInterface
```

Comment:
A pattern we have been moving to us breaking all of the tests out of the `_test` package and placing them on the same package as that of which is being tested.

Should we just rip the band aid off in this PR and give the same treatment to the tests in this package?

Reply:
The initial purpose of moving mocks was to move those tests out of _test package, but the legacy tests have apiserver package imports which cause import cycle issue.
So I had to revert back to keep the _test package.
I believe we will fix this once we remove the state stuff entirely.